libkineto/include/ActivityType.h

```diff
@@ -22,6 +22,7 @@ enum class ActivityType {
     CPU_INSTANT_EVENT, // host side point-like events
     PYTHON_FUNCTION,
     OVERHEAD, // CUPTI induced overhead events sampled from its overhead API.
+    HPU_OP,
```

Comment:
please add a comment

Reply:
Oh also do you intend for this to be always on by default, if not you can move it below the GLOW_RUNTIME, this will make it an optional activity type

Reply:
Hi @briancoutinho. Can you please explain what it means for an activity to be on by default? I expect to always see HPU_OP activity type when user selects torch.profiler.ProfilerActivity.HPU activity on HPU-enabled environment. But without this it shouldn't be produced. Does this mean the activity is on by default?

Reply:
sorry about the delay:( oh the default activities are in case the trace is captured asynchronously using a configuration file (on-demand profiling mode). The torch.profiler version should not be dependent on this